source/exploring_catalog_and_datasets/04_getting_involved/sharing_through_social_networks.rst

```diff
@@ -0,0 +1,42 @@
+Sharing through social networks
+===============================
+
+.. admonition:: Important
+   :class: important
+
+   Since shortcuts to social networks are an optional feature, they are not activated on every OpenDataSoft portal. The activation of shortcuts to social networks entirely depends on the portal administrators.
```

Comment:
> The activation of shortcuts to social networks entirely depends on the portal administrators.

The activation of shortcuts to social networks **depends entirely** on the portal administrators.